spark_pipeline_framework/transformers/fhir_receiver/v1/fhir_receiver.py

```diff
@@ -472,6 +484,17 @@ def _transform(self, df: DataFrame) -> DataFrame:
             self.refresh_token_function
         )
 
+        if parameters and parameters.get("flow_name"):
+            user_agent_value = (
+                f"{parameters['team_name']} : {parameters['flow_name']}"
```

Comment:
should be 
 f"{parameters['team_name']}:helix.pipelines:{parameters['flow_name']}"